Repository/Vcs/GitDriver.php

```diff
@@ -35,4 +36,24 @@ public function getComposerInformation($identifier)
 
         return ProcessUtil::getComposerInformation($this->cache, $this->infoCache, $this->repoConfig['asset-type'], $this->process, $identifier, $resource, sprintf('git show %s', $resource), sprintf('git log -1 --format=%%at %s', escapeshellarg($identifier)), $this->repoDir, '@');
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function initialize()
+    {
+        if (isset($this->config->get('fxp-asset')['git-driver']['skip-update'])) {
```

Comment:
You cannot use a hierarchy in the config to use environment variables (`FXP_ASSET__GIT_SKIP_UPDATE` env variable is mapped with the `config.fxp-asset.git-skip-update` package configuration).

You use the Composer config and not the [Fxp Asset Config](https://github.com/fxpio/composer-asset-plugin/blob/master/Config/Config.php) (I cannot extend the Composer config).

To access the Fxp asset config from the `GitDriver::initialize()` method, you must get the instance of `AssetRepositoryManager` stored in the `repoConfig` property and call the `getConfig()` method:

```php
/* @var AssetRepositoryManager $arm */
$arm = $this->repoConfig['asset-repository-manager'];

if (null !== ($skip = $arm->getConfig()->get('git-skip-update'))) {
    // your code ... > strtotime($skip)
}
```

So, replace this `if` condition by this snippet.